apps/site/lib/site_web/templates/trip_plan/_to_from_inputs.html.eex

```diff
@@ -30,15 +31,18 @@
     <div id="trip-plan__search--from__results" class="c-search-bar__results"></div>
     <div id="trip-plan__location-error--from" role="alert" aria-live="assertive" tabindex="0" class="c-trip-plan-widget__location-error location-error"><%= @from_error %></div>
   </div>
-
+  <div id="trip-plan-reverse-control" class="c-trip-plan-widget__reverse-control-vert">
+    <%= fa "long-arrow-down" %>
+    <%= fa "long-arrow-up" %>
+  </div>
+  <%= content_tag(:span, "(Required)", id: "trip-plan__required--to", class: "m-trip-plan__required#{if @to_error === "", do: " m-trip-plan__hidden"}") %>
+  <div id="trip-plan-reverse-control" class="c-trip-plan-widget__reverse-control">
+    <%= fa "long-arrow-down" %>
+    <%= fa "long-arrow-up" %>
+  </div>
   <div id="trip-plan__search--to" class="c-search-bar">
     <label for="to" class="c-search-bar__header c-trip-plan-widget__label--to" id="trip-plan__label--to">
       To&nbsp;
-      <%= content_tag(:span, "(Required)", id: "trip-plan__required--to", class: "m-trip-plan__required#{if @to_error === "", do: " m-trip-plan__hidden"}") %>
```

Comment:
I think should stay where it is and the additional one added above, otherwise it's going to affect other instances of this form.